repositories/strategies/spacedOrderStrategy.js

```diff
@@ -60,8 +60,10 @@ const getOrUpdateOrderForInsert = async (
     return maxOrder;
   }
 
+  collection.splice(position, 0, { id: "dummyElement" });
```

Comment:
Maybe this work could be done as an abstract util for array moving so it can be tested as a pure function

Reply:
I do agree here but after looking at it I'm not sure it would work as a pure function as it requires the use of database functions when the values on either side converge. We could pull that function out and then make the rest a pure function but I feel that might make a already complicated function even more complex?

Reply:
Yeah I was hoping to make just the array manipulation pure. So it would prep the data ready but if that’s not possible it can wait as it isn’t too bad.